admin_guide/assembly_replace-etcd-member.adoc

```diff
@@ -0,0 +1,28 @@
+[[assembly_replace-etcd-member]]
+= Replacing a failed etcd member
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: replace-etcd-member
+
+toc::[]
+
+
+In cases where etcd members have failed and you still have a quorum of etcd
+cluster members running, you can use the surviving members to
+add additional etcd members without downtime.
+
+include::admin_guide/topics/proc_removing-failed-etcd-member.adoc[leveloffset=+1]
+
+[[adding_an_etcd_node]]
+== Adding an etcd member
+
+You can add an etcd host either by using an Ansible playbook or by manual steps.
```

Comment:
Can I suggest "You can add an etcd host using an Ansible playbook, or using manual steps."

It's a nit :P

Reply:
IBM Style actually uses a similar "using/by using" case as its example for when to add prepositions to improve clarity, so I'm not going to take that suggestion.

Can you show me what style guide uses that extra comma? (I'm really curious. I've seen commas used like that in several places in our collection, but I can't find a reference for that usage. Based on the comma usage that I learned, that comma's unnecessary. English isn't static, though, so I want to see if it's changed.)